apps/app.go

```diff
@@ -98,9 +99,13 @@ const (
 // Function describes app's function mapping
 // For now Function can be either AWS Lambda or HTTP function
 type Function struct {
+	Path    string `json:"path"`
 	Name    string `json:"name"`
 	Handler string `json:"handler"`
 	Runtime string `json:"runtime"`
+
+	// filled out by appsctl
+	LambdaName string `json:"lambda_name,omitempty"`
```

Comment:
is the assignment part of this PR?

Reply:
This will make manifest file mutable right? I'm 1/5 that if possible we should stick with the immutable manifest.

Reply:
What is the difference between `Name` and `LambdaName`?

Reply:
1. `Name` - my understanding is that's the name of an entrypoint that AWS will use to invoke a function. `LambdaName` is the **invokable** name, the instance only needs to know `Path`->`LambdaName` and does not need anything else.
2. There really are 2 different things, both described as "the manifest":
  - a. a declaration of metadata and assets by the developer
  - b. a complete, "actionable" manifest generated by `appsctl`. The latter can be different from the former. 
Why is immutability of the manifest important? I'd much rather see the functions themselves immutable, i.e. content-addressed.

Reply:
Maybe I'm missing something, but `Name`  is what we use right now to invoke the lambda function (https://github.com/mattermost/mattermost-plugin-apps/blob/6764b4cdaf4628e7f0e87236e58d5c74ed134989/server/api/impl/upstream/upawslambda/upstream.go#L64)

Reply:
Removed `LambdaName`